backend/src/test/java/ca/bc/gov/hlth/hnweb/BaseControllerTest.java

```diff
@@ -73,4 +82,22 @@ protected Transaction assertTransactionCreated(TransactionType type) {
 		return opt.get();
 	}
 	
+	/**
+	 * Asserts the the Transaction is created.
```

Comment:
Seems typo. Remove extra the

Reply:
The entire comment was incorrect so updated it all, fixed typo in assertTransactionCreated method description